src/test/java/uk/ac/ebi/eva/pipeline/io/readers/VcfHeaderReaderTest.java

```diff
@@ -48,6 +51,20 @@
     @Rule
     public PipelineTemporaryFolderRule temporaryFolderRule = new PipelineTemporaryFolderRule();
 
+    @Test(expected = IOException.class)
+    public void testDuplicates() throws Exception {
```

Comment:
`testDuplicateSamples` to make clear that you are not referring to, for instance, duplicate records.